api/v1/views/places.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/python3
+from api.v1.views import app_views
+from flask import Flask, jsonify, abort, request
+from models import storage
+from models.place import Place
+from models.city import City
+
+
+@app_views.route('/cities/<city_id>/places',
+                 methods=['GET'], strict_slashes=False)
+def get_places_cities_id(city_id):
+    """ Return the places related to a state id """
+    catch_city = storage.get('City', city_id)
+    match_places = storage.all('Place')
+    if catch_city is None:
+        abort(404)
+    places_list = []
+    for place in match_places.values():
+        if place.city_id == city_id:
+            places_list.append(place.to_dict())
+    return jsonify(places_list)
+
+
+@app_views.route('/places/<place_id>', methods=['GET'], strict_slashes=False)
+def get_place_id(place_id):
+    """ Return the place form a id """
+    catch_place = storage.get('Place', place_id)
+    if catch_place is None:
+        abort(404)
+    return jsonify(catch_place.to_dict())
+
+
+@app_views.route('/places/<place_id>',
+                 methods=['DELETE'], strict_slashes=False)
+def del_place_id(place_id):
+    """ Delete the place form a id """
+    catch_place = storage.get('Place', place_id)
+    if catch_place is None:
+        abort(404)
+    storage.delete(catch_place)
+    storage.save()
+    return jsonify({}), 200
+
+
+@app_views.route('/cities/<city_id>/places',
+                 methods=['POST'], strict_slashes=False)
+def post_places(city_id):
```

Comment:
saw in other scripts that this definition could use a better name. Something like "post_cities_id_places"

Reply:
Change fixed